src/libraries/Microsoft.Extensions.HostFactoryResolver/src/Microsoft.Extensions.HostFactoryResolver.Sources.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <!-- The project doesn't compile anything therefore create the package during build. -->
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild Condition="'$(BuildingAnOfficialBuildLeg)' != 'true' or '$(BuildAllConfigurations)' == 'true'">true</GeneratePackageOnBuild>
```

Comment:
Will the `BuildAllConfigurations` part of this be redundant?  Doesn't the all configurations leg still seperately pack all csprojs? 
https://github.com/dotnet/runtime/blob/c1ed0db3307cd9b74ee77c2f8c93b7f129d86f5c/src/libraries/libraries-packages.proj#L16-L17

Reply:
You are right :) Let me clean that up.